index.js

```diff
@@ -203,6 +205,7 @@ module.exports = function (config) {
 
   if (koop.config.export_workers) {
     var kue = require('kue')
+
     koop.Exporter.export_q = kue.createQueue({
```

Comment:
agree, Exporter is used in BaseModel too. Should be pulled out into a module (koop-exporter) and used only where necessary. Should also be lowercased as it's not a class/constructor.